controllers/storagecluster/odfinfoconfig.go

```diff
@@ -137,6 +137,14 @@ func getOdfInfoData(r *StorageClusterReconciler, storageCluster *ocsv1.StorageCl
 		return "", err
 	}
 
+	annotations := map[string]string{}
+	for key, value := range storageCluster.GetAnnotations() {
+		parts := strings.Split(key, "/")
+		if len(parts) == 2 && strings.HasSuffix(parts[0], "ocs.openshift.io") {
```

Comment:
Just a note: As a side effect this will also include other annotations like "multicluster.ocs.openshift.io", if any.